named-entity-recognition/README.md

```diff
@@ -13,16 +13,34 @@ There are some sources for labeled NER data that can be used to train systems.
 
 You can also create data yourself if it doesn't exist.
 
-TODO: A good procedure to do this should be filled in.
+* Start by collecting unlabelled texts from popular domains like News articles or Wikipedia. If unlabelled texts do not exist, you can translate some sentences. 
```

Comment:
News
-> news

you can translate some sentences
-> you can translate some sentences, but you should try to get target-language text if at all possible because translated text can be unnatural